src/base/system_parameter.c

```diff
@@ -3798,17 +3792,6 @@ static SYSPRM_PARAM prm_Def[] = {
    (char *) NULL,
    (DUP_PRM_FUNC) NULL,
    (DUP_PRM_FUNC) NULL},
-  {PRM_ID_HA_UNACCEPTABLE_PROC_RESTART_TIMEDIFF_IN_MSECS,
-   PRM_NAME_HA_UNACCEPTABLE_PROC_RESTART_TIMEDIFF,
-   (PRM_FOR_CLIENT | PRM_FOR_HA | PRM_TIME_UNIT),
```

Comment:
It is better to mark parameters as obsolete instead of removing them completely.

Reply:
Fixed